README.md

```diff
@@ -1 +1,50 @@
 # Operate First in Action! 
+
+Click on the pic below! and watch our video from Red Hat Summit 2022 and see Operate First in Action! _Note: this is WIP and image links to different video._
+
+[![video thumbnail](assets/images/opf-in-action-thumbnail.png)](https://youtu.be/v4mur-_ywBM)
+https://youtu.be/v4mur-_ywBM
```

Comment:
I guess this will be our interview video? Are we allowed to publish it before it is shown in the summit?
Not a blocker for this PR, just a question.

Reply:
Yes, I put in a place holder video as noted in the text, to make sure it worked correctly. I assume we should probably wait till summit to upload the actual video.